src/Pages/Preferences/Preferences.tsx

```diff
@@ -289,6 +291,61 @@ const Preferences = () => {
                 )}
               </div>
             </Flex>
+            <Flex className="preference-option">
+              <div className="preference-title">{t('PREF_RCON_PORT')}</div>
+              <TextInput
+                type={'input'}
+                defaultValue={settings?.internal.rconPort}
+                onLeave={(e) => {
+                  try {
+                    const port = Number.parseInt(e);
+                    handleSettingChange('rconPort', port, 'internal');
+                    //eslint-disable-next-line no-empty
+                  } catch {}

```

Comment:
Might be better to use `isNaN` instead of a try catch here

```
const port = Number.parseInt(e);
if (!isNaN(port)) {
  handleSettingChange('rconPort', port, 'internal');
}
```